### DISCUSSION TITLE
In-story choices feasibility with the Lunii device?

### DISCUSSION THREAD

**Discussion by akelai** [marian-m12l/studio#185] [Ideas]
Would it be feasible (with the Lunii device), and some homebrew software, maybe based on this one) to create the possibility of stories with choices during the story, thus changing the narration flow?

E.g. "_...The girl had a yellow door in front of her, but she could ignore it and proceed on her way_".

So one can select the choice, and the story goes on taking that into account.
Then at some point there's another choice offered.

Just like those books with this "feature", where you jump at some page based on the choice.

**Comment by vpoulailleau** [comment]
It's feasible with the Lunii device, some official story packs do that.

I've never done that with STUdio, but I don't see any difficulty to do it (maybe I'm wrong, I haven't tried).

**Reply by akelai** [reply]
Thanks, that's interesting!

**Reply by marian-m12l** [reply]
Indeed, that's possible, you can chain a menu node after a story node for instance.

**Reply by akelai** [reply]
That sounds exciting. I'll definitely try!

**Reply by darrachequesne** [reply]
Hi! I'd like to create a "choose your own adventure" story, so I'd need to store the choices of the user (menu node) and reuse it later (but not immediately).

@marian-m12l is it currently possible?

@vpoulailleau I've never encountered an official story with this feature, could you please provide an example?
